src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/data/OffenderNonAssociation.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.data
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Offender non-association")
+class OffenderNonAssociation(
+  @Schema(description = "The offenders number", example = "G0135GA", required = true)
+  val offenderNo: String
```

Comment:
we tend to refer to  this as `prisonNumber` in the frontend and have started using it more through the codebase